docker-compose-firecrawl.yml

```diff
@@ -2,18 +2,20 @@ name: firecrawl
 services:
   # Firecrawl services
   playwright-service:
-    image: trieve/playwright-service-ts:latest
+    image: trieve/puppeteer-service-ts-v0.0.6
```

Comment:
Is this a new docker image or a version bump?

Reply:
New image